src/index.ts

```diff
@@ -83,6 +86,7 @@ export class EppoJSClient extends EppoClient {
     subjectAttributes?: Record<string, any>,
     assignmentHooks?: IAssignmentHooks,
   ): string | null {
+    EppoJSClient.getAssignmentInitializationCheck();
```

Comment:
I think we _do_ want to warn people who request assignments on uninitialized instances.